vpr/src/place/noc_place_utils.h

```diff
@@ -65,7 +65,8 @@ void initial_noc_routing();
  *
  * @param costs Used to get aggregate bandwidth and latency costs.
```

Comment:
Need to comment the second parameters, and the special case of an empty vector for it.